code/game/objects/structures/hybrisa_props.dm

```diff
@@ -31,7 +35,8 @@
 		if(0 to 500)
 			icon_state = icon_state = "[initial(icon_state)]_damage_[5]"
 
-/obj/structure/prop/urban/vehicles/proc/explode(dam, mob/M)
+///spawns a bunch of debris and plays a sound when a vehicle is wrecked
+/obj/structure/prop/urban/vehicles/proc/explode()
```

Comment:
Doesn't actually blow up, sad

Reply:
It now has a 10% chance of blowing up when destroyed

Reply:
basado